EDMF_Environment.pxd

```diff
@@ -32,7 +32,7 @@ cdef class EnvironmentVariables:
         EnvironmentVariable THVvar
         Grid Gr
         bint use_tke
```

Comment:
To avoid confusion over the connotations of "use" vs. "calc", should "use_tke" be changed to "calc_tke" consistent with the scalar variance convention? It is not a 100% clean analogy, but there is the option to calculate the TKE equation but get eddy diffusivities from the similarity profile. In that case, we should check "use_tke"/"calc_tke" is true whenever the similarity diffusivity flag is set to false

Reply:
OK, makes sense. Do you want it done before or after merging with pull #5 ?

Reply:
If you don't mind, could you do it now?

Reply:
OK